### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by Lin-Buo-Ren**
Build fails in my Ubuntu 16.04 LXD container, this is a series of patches that fix the encountered issues.

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -957,8 +957,8 @@ parts:
     - --disable-static
     - --with-cairo=no
     override-build: |
-      apt-get remove -yyq libfreetype6
-      ln -s ../local/lib/libfreetype.so.6 $SNAPCRAFT_STAGE/usr/lib/
+      sudo apt-get remove -yyq libfreetype6
+      ln -fs ../local/lib/libfreetype.so.6 $SNAPCRAFT_STAGE/usr/lib/
```
Comment (diddledan) [inline snap/snapcraft.yaml]: :+1:

File: snap/snapcraft.yaml [hunk 2]
```diff
@@ -297,7 +297,7 @@ parts:
   freetype-prebuild:
     after:
     - fix-pkgconfig-files
-    source: https://download.savannah.gnu.org/releases/freetype/freetype-2.9.1.tar.bz2
+    source: http://download.savannah.nongnu.org/releases/freetype/freetype-2.9.1.tar.bz2
```
[the remark this answers is not in the document]
Comment (Lin-Buo-Ren) [reply snap/snapcraft.yaml]: @diddledan At the time I run the build the former URL timeout-ed, as it is no longer the case now I'll drop this patch.

File: snap/snapcraft.yaml [hunk 3]
```diff
@@ -957,8 +957,8 @@ parts:
     - --disable-static
     - --with-cairo=no
     override-build: |
-      apt-get remove -yyq libfreetype6
-      ln -s ../local/lib/libfreetype.so.6 $SNAPCRAFT_STAGE/usr/lib/
+      sudo apt-get remove -yyq libfreetype6
```
[the remark this answers is not in the document]
Comment (Lin-Buo-Ren) [reply snap/snapcraft.yaml]: But if we don't use sudo the command will fail instantly in LXD due to insufficient privilege... :-/

Is there any way to go around the in-system libfreetype6 without removing it?

**Comment by Lin-Buo-Ren** [comment]
I've made the requested changes, please review.

File: snap/snapcraft.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (diddledan) [reply snap/snapcraft.yaml]: I don't think we can avoid removing `freetype6`. It confuses everything by having it installed, but some build dependencies pull it in so we need to forcibly remove it again.

**Pull request merged**